content/docs/1.1.0/advanced-resources/data-recovery/recover-without-system.md

```diff
@@ -11,20 +11,49 @@ This command gives users the ability to restore a backup to a `raw` image or a `
 
 3. Specify the host path of output file by modifying field `hostpath` of volume `disk-directory`. By default the directory is `/tmp/restore/`.
 
-4. Set the first argument (backup url) by replacing `<BACKUP_URL>`, e.g. `s3://backupbucket@us-east-1/backupstore?backup=backup-bd326da2c4414b02&volume=volumeexamplename`. Do not delete `''`.
+4. Set the first argument (backup url) by replacing `<BACKUP_URL>`, e.g. `s3://<your-bucket-name>@<your-aws-region>/backupstore?backup=<backup-name>&volume=<volume-name>`. Do not delete `''`.
+
+    - `<backup-name>` and `<volume-name>` can be retrieved from backup.cfg stored in the backup destination folder, e.g. `backup_backup-72bcbdad913546cf.cfg`. The content will be like below: 
+
+        ```json
+        {"Name":"backup-72bcbdad913546cf","VolumeName":"volume_1","SnapshotName":"79758033-a670-4724-906f-41921f53c475"......}
```

Comment:
nit: Minor formatting problem.
<img width="954" alt="Screenshot 2022-11-01 at 12 56 53 PM" src="https://user-images.githubusercontent.com/60111667/199326614-1cdc5029-aa48-49d7-bdbe-36a724420f76.png">

Reply:
I had fixed it. Thank you.